### DISCUSSION TITLE
Task suggestions for Sign Up team

### DISCUSSION THREAD

**Discussion by FabiolaZTorres** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#58] [Ideas]
Hello team!

As we continue working on the Sign-Up page, it's important we make sure to cover all necessary features to ensure the best user experience and security. I think it would be great if we could use this space to gather ideas and suggestions for features and tasks we could add to improve functionality.

Thanks in advance for your ideas!

**Comment by PerezMarcelo** [comment]
1. Email confirmation

- Verify ownership
- Prevent spam accounts
- Security
- Password recovery

2. reCAPTCHA

- Prevent automated attacks
- Reduce spam
- Improve security
- Protect resources
